packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/tableFeatures.ts

```diff
@@ -151,6 +152,23 @@ const UpDownInTable: BuildInEditFeature<PluginKeyboardEvent> = {
     defaultDisabled: !Browser.isChrome && !Browser.isSafari,
 };
 
+/**
+ * Requires @see ExperimentalFeatures.DeleteTableWithBackspace
+ * Delete a table selected with the table selector pressing Backspace key
+ */
+const DeleteTableWithBackspace: BuildInEditFeature<PluginKeyboardEvent> = {
+    keys: [Keys.BACKSPACE],
+    shouldHandleEvent: (event: PluginKeyboardEvent, editor: IEditor) =>
+        cacheIsWholeTableSelected(event, editor) &&
+        editor.isFeatureEnabled(ExperimentalFeatures.DeleteTableWithBackspace),
```

Comment:
Should we check `editor.isFeatureEnabled(ExperimentalFeatures.DeleteTableWithBackspace),` first? Since it could take less time that trying to create a VTable and check if it is selected